packages/lit-virtualizer/src/lib/scroll.ts

```diff
@@ -0,0 +1,57 @@
+import { directive, NodePart, TemplateResult } from 'lit-html';
+import { VirtualScroller } from './uni-virtualizer/lib/VirtualScroller';
+import { Layout } from './uni-virtualizer/lib/layouts/Layout';
+import { LitMixin } from './repeat';
+
+export const LitScroller = LitMixin(VirtualScroller);
+
+// scrollers are type: any because of the Mixin problem, described at LitMixin
+// in repeat.ts.
+const partToScroller: WeakMap<NodePart, any> = new WeakMap();
+
+/**
+ * Configuration options for the scroll directive.
+ */
+interface ScrollConfig {
+  // A function that returns a lit-html TemplateResult. It will be used
+  // to generate the DOM for each item in the virtual list.
+  template?: (item: any, index?: number) => TemplateResult,
+
+  layout?: Layout,
```

Comment:
Think we need to document what layouts are, probably a little here, and in https://github.com/PolymerLabs/uni-virtualizer/tree/master/packages/lit-virtualizer